app/web/src/newhotness/Explore.vue

```diff
@@ -451,6 +473,25 @@ const actionViewList = computed(
   () => actionViewListRaw.data.value?.actions ?? [],
 );
 
+const componentsWithFailedActions = computed(() => {
```

Comment:
These could be combined into one to prevent two O(n) walks. Probably an interface with two sets.

Marking for the next PR. I don't think it blocks this one.